### ISSUE TITLE
What to do with Extra items in the target which are not in English Source notes?

### ISSUE THREAD

**Issue by mannycolon** [unfoldingWord-box3/tn-converter-app#51]
Add blank fields for the source English columns to allow the GL to show extra TN quotes

**Comment by mannycolon** [comment]
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/02/20.md ???

**Comment by mannycolon** [comment]
# More GL notes than English notes

- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/02/20.md 
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/03/04.md
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/04/06.md
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/04/19.md
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/04/21.md
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/05/07.md
- https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/1jn/05/17.md

**Comment by elsylambert** [comment]
Looking at Russian Notes for 3 John. https://git.door43.org/ru_gl/ru_tn_1lv/src/branch/master/3jn/01/12.md
Atleast one of the Notes is not rendered in each verse from verse 8 to 15. The line is blank on the GL side.
![Screen Shot 2020-10-02 at 11.25.38 AM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/2ee13ca4-e4d4-4adc-aebc-34fd81df0359)


This happened when I loaded my existing 3Jn project.

I started a new 3 Jn Project and it did an overwrite and so the data in columns look fine. All the extra notes are rendered.
